src/SwapServiceProvider.php

```diff
@@ -99,9 +99,11 @@ private function getConfigPath($path = '')
     private function getSimpleCache()
     {
         if ($cache = $this->app->config->get('swap.cache')) {
-            $store = $this->app['cache']->store($cache)->getStore();
+            $store = $this->app['cache']->store($cache);
 
-            return new SimpleCacheBridge(new IlluminateCachePool($store));
+            return $store instanceof \Psr\SimpleCache\CacheInterface
+                ? $store
+                : new SimpleCacheBridge(new IlluminateCachePool($store->getStore()));
```

Comment:
Can you throw an exception if the class does not exist with a message suggesting to install the php-cache package ?

Reply:
A added changes in order to throw an exception if PSR-6 adapter or PSR-16 bridge is missing.